<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>501</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="find">
    <attribute name="title">
     <string>Find</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>15</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findPrevious">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>272</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
     <property name="icon">
      <iconset theme="go-up"/>
     </property>
    </widget>
    <widget class="QPushButton" name="selectAll">
     <property name="geometry">
      <rect>
       <x>337</x>
       <y>273</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
    <widget class="QComboBox" name="searchLineEdit">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>12</y>
       <width>431</width>
       <height>25</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="advancedOptions">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>48</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show advanced options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findNext">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>271</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
     <property name="icon">
      <iconset theme="go-down"/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxAdvanced">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>80</y>
       <width>461</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Advanced</string>
     </property>
     <widget class="QRadioButton" name="searchPlainText">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>32</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search &amp;plain text</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="searchRegularExpression">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>60</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search with &amp;regular expressions</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="searchSpecialCharachters">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>89</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search &amp;with special characters (\n, \r, \t, \0, \u...,\x...)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="matchWholeWord">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>120</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Match whole word only</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="matchCase">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>150</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Match case</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="replace">
    <attribute name="title">
     <string>Replace</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="advancedSearch">
    <attribute name="title">
     <string>Advanced search</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
